code/modules/mob/living/basic/clown/clown.dm

```diff
@@ -8,191 +8,114 @@
 	icon_gib = "clown_gib"
 	health_doll_icon = "clown" //if >32x32, it will use this generic. for all the huge clown mobs that subtype from this
 	mob_biotypes = MOB_ORGANIC|MOB_HUMANOID
-	turns_per_move = 5
 	response_disarm_continuous = "gently pushes aside"
 	response_disarm_simple = "gently push aside"
 	response_harm_continuous = "robusts"
 	response_harm_simple = "robust"
-	speak = list("HONK", "Honk!", "Welcome to clown planet!")
-	emote_see = list("honks", "squeaks")
-	speak_chance = 1
 	combat_mode = TRUE
 	maxHealth = 75
 	health = 75
-	speed = 1
-	harm_intent_damage = 8
 	melee_damage_lower = 10
 	melee_damage_upper = 10
 	attack_sound = 'sound/items/bikehorn.ogg'
-	obj_damage = 0
+	attacked_sound = 'sound/items/bikehorn.ogg'
 	environment_smash = ENVIRONMENT_SMASH_NONE
-	del_on_death = 1
-	loot = list(/obj/effect/mob_spawn/corpse/human/clown)
+	basic_mob_flags = DEL_ON_DEATH
 	initial_language_holder = /datum/language_holder/clown
-	atmos_requirements = list("min_oxy" = 5, "max_oxy" = 0, "min_plas" = 0, "max_plas" = 1, "min_co2" = 0, "max_co2" = 5, "min_n2" = 0, "max_n2" = 0)
-	minbodytemp = 270
-	maxbodytemp = 370
+	habitable_atmos = list("min_oxy" = 5, "max_oxy" = 0, "min_plas" = 0, "max_plas" = 1, "min_co2" = 0, "max_co2" = 5, "min_n2" = 0, "max_n2" = 0)
+	minimum_survivable_temperature = 270
+	maximum_survivable_temperature = 370
 	unsuitable_atmos_damage = 10
 	unsuitable_heat_damage = 15
-	footstep_type = FOOTSTEP_MOB_SHOE
 	faction = list(FACTION_CLOWN)
+	ai_controller = /datum/ai_controller/basic_controller/clown
+	speed = 1.4 //roughly close to simpleanimal clowns
+	///what reagent do we inject on attack
 	var/attack_reagent
```

Comment:
Only two of them use this, why not just put the component on those subtypes instead?

Reply:
did